app/grants/views.py

```diff
@@ -1473,13 +1473,19 @@ def grant_edit(request, grant_id):
         if logo:
             grant.logo = logo
 
-        twitter_handle_1 = request.POST.get('handle1', None)
-        if twitter_handle_1:
-            grant.twitter_handle_1 = twitter_handle_1
+        twitter_handle_1 = request.POST.get('handle1', '').strip('@')
+        twitter_handle_2 = request.POST.get('handle2', '').strip('@')
 
-        twitter_handle_2 = request.POST.get('handle2', None)
-        if twitter_handle_2:
-            grant.twitter_handle_2 = twitter_handle_2
+        if twitter_handle_1 and not re.search(r'^@?[a-zA-Z0-9_]{1,15}$', twitter_handle_1):
```

Comment:
Chibie are these handling people pasting the url? because the main problem was people posting https://twitter.com/octavioamu

Reply:
take a look at this https://regex101.com/r/s6JoZ5/3